src/cmd/opt.js

```diff
@@ -1,17 +1,11 @@
 import { $init, tools } from '../../obj/wasm-tools.js';
 const { metadataShow, print } = tools;
 import { writeFile } from 'fs/promises';
-import { fileURLToPath } from 'url';
 import c from 'chalk-template';
-import { readFile, sizeStr, fixedDigitDisplay, table, spawnIOTmp, setShowSpinner, getShowSpinner } from '../common.js';
+import { readFile, sizeStr, fixedDigitDisplay, table, spawnIOTmp, setShowSpinner, getShowSpinner, require } from '../common.js';
 import ora from '#ora';
 
-let WASM_OPT;
-try {
-  WASM_OPT = fileURLToPath(new URL('../../node_modules/binaryen/bin/wasm-opt', import.meta.url));
-} catch {
-  WASM_OPT = new URL('../../node_modules/binaryen/bin/wasm-opt', import.meta.url);
-}
+const WASM_OPT = require.resolve('binaryen/bin/wasm-opt');
```

Comment:
It's better to use `import.meta.resolve` here, see https://github.com/bytecodealliance/jco/blob/main/src/cmd/run.js#L90C57-L90C76 for an example.

Reply:
Thanks for the pointer. I changed the PR to use `import.meta.resolve`, which passes all tests now, with one caveat: `import.meta.resolve` actually resolves to the source file being imported, not just the directory of the package, which requires us to traverse up to find it. This works correctly for the current inputs, and is an improvement to the current code, but we might need to reconsider if we need to handle more complex inputs.

Reply:
There is a [single windows-only test failure](https://github.com/bytecodealliance/jco/actions/runs/10607488754/job/29400051018?pr=493). I think it is unrelated to this PR (a race condition in a compiled binary), but I'm not able to repro locally. I will try rerunning the CI job.

Reply:
Rerun succeeded.